coresdk/src/coresdk/camera.h

```diff
@@ -330,6 +330,50 @@ namespace splashkit_lib
      * @attribute method center_on
      */
     void center_camera_on(sprite s, const vector_2d &offset);
+	/**
+	 * Set the camera view to be centered over a list of sprites. The offset
+	 * vector allows you to move the sprites from the direct center of the screen.
+	 *
+	 * @param s        A vector of sprites to track.
+	 * @param offset_x An additional offset added to the camera, allowing you to
+	 *                 position the sprites offset from the center of the screen.
+	 * @param offset_y An additional offset added to the camera, allowing you to
+	 *                 position the sprites offset from the center of the screen.
+	 */
+	void center_camera_on(vector<sprite> s, double offset_x, double offset_y);
+	/**
+	 * Set the camera view to be centered over an array of sprites. The offset
+	 * vector allows you to move the sprites from the direct center of the screen.
+	 *
+	 * @param s        An array of sprites to track.
+	 * @param size     The size of the sprite array.
+	 * @param offset_x An additional offset added to the camera, allowing you to
+	 *                 position the sprites offset from the center of the screen.
+	 * @param offset_y An additional offset added to the camera, allowing you to
+	 *                 position the sprites offset from the center of the screen.
+	 */
+	void center_camera_on(sprite s[],int size , double offset_x, double offset_y);
```

Comment:
I don't think the translator supports arrays - have you tested rebuilding the language translations. See the `tools/scripts/deploy.sh` script.